src/main/java/Deadline.java

```diff
@@ -0,0 +1,18 @@
+public class Deadline extends Task {
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+        System.out.println("    ____________________________________________________________");
```

Comment:
I would suggest putting the print statements in a separate function as there may be instances you are creating a Deadline object but you do not require to print these statements.